ql/indexes/indexmanager.cpp

```diff
@@ -26,15 +26,27 @@ namespace QuantLib {
     }
 
     const TimeSeries<Real>& IndexManager::getHistory(const std::string& name) const {
-        return data_[name].value();
+        return data_[name];
     }
 
     void IndexManager::setHistory(const std::string& name, TimeSeries<Real> history) {
         data_[name] = std::move(history);
```

Comment:
This should also notify explicitly, it's no longer an ObservableValue and assignment doesn't notify

Reply:
Yes. And we want to notify when we clear data as well I suppose?